### PR TITLE
Single article

### PR DESCRIPTION
**Pull request by olivershh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
   return (
     <div className="App">
-      <Header></Header>
-      <Routes></Routes>
+      <Header />
+      <Routes>
+        {/* <Route path="/" element={<Hub/>} */}
+        {/* <Route path="/random" element=??? */}
+
+        <Route path="/all" element={<All />} />
```
Comment (popatre) [inline src/App.js:15]: I would name this route better. all what? articles/all?

File: src/components/ArticleCardSmall.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export default function ArticleCardSmall({article}) {
+    console.log(article)
```
Comment (popatre) [inline src/components/ArticleCardSmall.jsx:2]: remove log

File: src/App.js (hunk 1 above)
Comment (popatre) [reply src/App.js:15]: Also, the component name might get confusing, as you'll most likely reuse it for the filtered articles as well

File: src/components/site-wide/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default function Navbar() {
+    return (
+        <nav className="nav-site-wide">
+            <a href="#">All</a>
+             <a href="#">Random</a>
+             <a href="#">Trending</a>
+             <a href="#">Post</a>
+        </nav>
```
Comment (popatre) [inline src/components/site-wide/Navbar.jsx:8]: These links dont work - you need to use the react router dom link element, to get it to change the url

File: src/pages/All.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react"
+import { getAllArticles } from "../components/api-calls"
+import ArticleCardSmall from "../components/ArticleCardSmall"
+
+export default function All() {
```
Comment (popatre) [inline src/pages/All.jsx:5]: Name component better - ArticleGrid? SOmething like that?

File: src/pages/All.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+            setArticles(articles)
+            setIsLoading(false)
+        })
+      
+    }, [])
+
+    return (
+        <main className="articles-list-main">
+        {articles.map(article => {
+            console.log(article)
```
Comment (popatre) [inline src/pages/All.jsx:23]: remove log

**Review by popatre — CHANGES_REQUESTED** [review]

File: src/pages/All.jsx (hunk 5 above)
Comment (olivershh) [reply src/pages/All.jsx:23]: done

File: src/App.js (hunk 1 above)
Comment (olivershh) [reply src/App.js:15]: Makes sense. Amended to ArticlesList

File: src/components/site-wide/Navbar.jsx (hunk 3 above)
Comment (olivershh) [reply src/components/site-wide/Navbar.jsx:8]: Sorted thanks

**Pull request merged**